src/handlers.js

```diff
@@ -380,30 +424,54 @@ async function jobHandler(message) {
     await this.createTasks({scopes: graphConfig.scopes, tasks: graphConfig.tasks});
   } catch (e) {
     debug('Creating tasks failed! Leaving comment on Github.');
-    groupState = 'failure';
+    groupState = {
+      status: 'completed',
+      conclusion: 'failure',
+      completed_at: new Date().toISOString(),
+    };
     await this.createExceptionComment({instGithub, organization, repository, sha, error: e});
   } finally {
-    debug(`Trying to create status for ${organization}/${repository}@${sha} (${groupState})`);
+    debug(`Trying to create check runs for ${organization}/${repository}@${sha} (${groupState})`);
     let eventType = message.payload.details['event.type'];
-    let statusContext = `${this.context.cfg.app.statusContext} (${eventType.split('.')[0]})`;
-    let description = groupState === 'pending' ? `TaskGroup: Pending (for ${eventType})` : 'TaskGroup: Exception';
-    await instGithub.repos.createStatus({
-      owner: organization,
-      repo: repository,
-      sha,
-      state: groupState,
-      target_url: INSPECTOR_URL + taskGroupId,
-      description,
-      context: statusContext,
-    });
+
+    await Promise.all(graphConfig.tasks.map(async (task, i) => {
+      const checkRun = await instGithub.checks.create(Object.assign( // TODO: need spread syntax
```

Comment:
++ to spread syntax :)

Reply:
I opened a PR for that: https://github.com/taskcluster/eslint-config-taskcluster/pull/7